changelogs/fragments/8373-honour-disable-logs.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - opentelemetry callback plugin - honour the disable_logs to avoid storing task results since they are not used regardless (https://github.com/ansible-collections/community.general/pull/8133).
```

Comment:
```suggestion
  - opentelemetry callback plugin - honour the ``disable_logs`` option to avoid storing task results since they are not used regardless (https://github.com/ansible-collections/community.general/pull/8133).
```